airbyte-integrations/connectors/source-okta/unit_tests/test_source.py

```diff
@@ -0,0 +1,126 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from unittest.mock import MagicMock
+
+from airbyte_cdk.sources.streams.http.auth import TokenAuthenticator
+from source_okta.source import (
+    CustomRoles,
+    GroupMembers,
+    GroupRoleAssignments,
+    Groups,
+    Logs,
+    OktaOauth2Authenticator,
+    SourceOkta,
+    UserRoleAssignments,
+    Users,
+)
+from unit_tests import (
```

Comment:
Use `pytest.fixture` decorator to reuse some values.

Reply:
done